File: challenge-105/ziameraj16/java/NthRoot.java
import java.util.Scanner;

public class NthRoot {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter value of N");
        double N = scanner.nextDouble();
        System.out.println("Enter value of k");
        int k = scanner.nextInt();
        System.out.printf("%.2f", findNthRoot(N, k));
    }

    public static double findNthRoot(double N, int k) {
        return Math.pow(k, 1/N);
    }
}

File: challenge-105/ziameraj16/kotlin/NthRoot.kt
import kotlin.jvm.JvmStatic
import java.util.*

object NthRoot {
    @JvmStatic
    fun main(args: Array<String>) {
        val scanner = Scanner(System.`in`)
        println("Enter value of N")
        val N = scanner.nextDouble()
        println("Enter value of k")
        val k = scanner.nextInt()
        System.out.printf("%.2f", findNthRoot(N, k))
    }

    fun findNthRoot(N: Double, k: Int): Double {
        return Math.pow(k.toDouble(), 1 / N)
    }
}
